Source/ACE.Database/ShardDatabase.cs

```diff
@@ -702,7 +702,6 @@ public List<Character> GetCharacters(uint accountId, bool includeDeleted)
                 .Include(r => r.CharacterPropertiesQuestRegistry)
                 .Include(r => r.CharacterPropertiesShortcutBar)
                 .Include(r => r.CharacterPropertiesSpellBar)
-                .Include(r => r.CharacterPropertiesSquelch)
```

Comment:
bad merge?

Reply:
Yes, thank you... Will try to clear that up here.